### DISCUSSION TITLE
51% attack？？

### DISCUSSION THREAD

**Discussion by AZ663** [Chia-Network/chia-blockchain#3288] [General]
![3339](https://user-images.githubusercontent.com/83150987/116768670-02323700-aa6b-11eb-9774-450150052f27.jpg)

Almost 40%

**Comment by keliew** [comment]
Chia is not your usual blockchain/crypto. Don't be worried about things that are already mentioned and thought of. See [Chia Consensus](https://docs.google.com/document/d/1tmRIb7lgi4QfKkNaxuKOBHRmwbVlGL4f7EsBDr_5xZE/edit#heading=h.kal110ww885w).
